DSpace/DSpace
# Features endpoint: Edit Collection Permissions

## References
* Fixes https://github.com/DSpace/DSpace/issues/2944

## Description
We've added REST Authorization Features:
- canManageAdminGroup
- canManageSubmitterGroup
- canManageTemplateItem
- canManageWorkflowGroup

These all describe whether or not the current user can manage the respective group for the Collection

## Instructions for Reviewers

List of changes in this PR:
* Added the four above-mentioned features for the Collection objects
* Provided tests for these new features

How to test this PR:
* Navigate to the {dspace.url}/server/api/authz/authorizations/search/object?uri={dspace.url}/server/api/core/collections/{collectionID}&embed=feature whilst being logged in as admin and verify that you can see the four above-mentioned features
* Repeat this for a normal EPerson, you won't see the features
* Repeat this for a collection admin EPerson, you'll see the features

## Checklist
